.github/workflows/control-slycot-src.yml

```diff
@@ -43,11 +32,10 @@ jobs:
         # Install compilers, libraries, and development environment
         sudo apt-get -y install gfortran cmake --fix-missing
         sudo apt-get -y install libblas-dev liblapack-dev
-        conda install -c conda-forge scikit-build setuptools-scm
 
         # Compile and install slycot
-        pip install -v --no-build-isolation --no-deps .
+        pip install -v .
 
     - name: Test with pytest
       working-directory: python-control
-      run: xvfb-run --auto-servernum pytest control/tests
+      run: pytest -v control/tests
```

Comment:
Do we want to use '-v' here?  It generates really long output in the logs.  (OK by me if it is useful, I just wasn't sure whether it was.)

Reply:
Yes, I prefer to see which tests including their parameters are actually run, skipped, or fail.